R-package/DESCRIPTION

```diff
@@ -24,7 +25,8 @@ Authors@R: c(
     person("Alberto", "Ferreira", role = c("ctb")),
     person("Daniel", "Lemire", role = c("ctb")),
     person("Victor", "Zverovich", role = c("cph")),
-    person("IBM Corporation", role = c("ctb"))
+    person("IBM Corporation", role = c("ctb")),
+    person("David", "Cortes", role = c("ctb"))
```

Comment:
proposing adding @david-cortes to DESCRIPTION, thanks to their excellent recent contributions (https://github.com/microsoft/LightGBM/pulls?q=is%3Apr+author%3Adavid-cortes+is%3Aclosed)